rclc/CHANGELOG.rst

```diff
@@ -1,6 +1,10 @@
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 Changelog for package rclc
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
+0.1.7 (2021-01-20)
+------------------
+* Corrected corrupted changelog
+
```

Comment:
Also something went wrong with your base version. Both ros2/rclc:master and micro-ros/rclc:master should have been updated to 0.1.7

Why is this change and the changes below in the other package.xml and CHANGELOG.rst files in this pull request?